src/store/reducers/weatherReducer.js

```diff
@@ -1,17 +1,13 @@
 import { actionTypes } from "../action-types";
 
-const initialState = {
-  cod: "",
-  message: 0,
-  cnt: 0,
-  list: [],
-  city: {},
-};
+const initialState = {};
```

Comment:
why removed this? There should be an initial state

Reply:
because the previously written keys for initial state were wrong.  So i have created an empty object where it gets updated when data is received from the API

Reply:
no, the initial state cant be empty.

Reply:
still not resolved